insights/specs/__init__.py

```diff
@@ -461,8 +461,8 @@ class Specs(SpecSet):
     ovs_vsctl_show = RegistryPoint()
     ovs_vswitchd_limits = RegistryPoint()
     pacemaker_log = RegistryPoint(filterable=True)
-    package_provides_java = RegistryPoint(multi_output=True)
-    package_provides_httpd = RegistryPoint(multi_output=True)
+    package_provides_java = RegistryPoint()
+    package_provides_httpd = RegistryPoint()
```

Comment:
@xiangce this change will break the existing code, I think you'll need to leave it as `multi_output`.

Reply:
@bfahr - yep, that was true. I reverted them. And added a new spec `package_provides_command` which works for any new commands. Please check it again. hanks.